tests/unit_tests/1001_lib_cpuset_validation_test.py

```diff
@@ -0,0 +1,117 @@
+import random
+import re
+
+
+from iocage_lib.ioc_json import IOCCpuset
+
+# For cpuset props we would like to test the following scenarios
+# 1) 0,1,2,3
+# 2) 0-3
+# 3) all
+# 4) off
+
+cpu_sets = IOCCpuset.retrieve_cpu_sets() + 1
+
+# Regex for point 1
+
+list_regex = re.compile(
```

Comment:
Tests should not have any condition logic, loops and, of course, regexes. Test should be as simple as possible, simple statements, e.g. `assert 1 + 1 == 2`.